dashboard/db/migrate/20170330214255_create_pd_program_registration.rb

```diff
@@ -0,0 +1,10 @@
+class CreatePdProgramRegistration < ActiveRecord::Migration[5.0]
+  def change
+    create_table :pd_facilitator_program_registrations do |t|
+      t.references :user, index: true
+      t.text :form_data
+
+      t.timestamps null: false
+    end
+  end
+end
```

Comment:
Why? Given that the model is dependent on this migration, shouldn't they be in the same PR?